README.rst

```diff
@@ -74,12 +74,10 @@ Install ``plone.restapi`` by adding it to your buildout.
 Python / Plone Compatibility
 ============================
 
-plone.restapi 8 requires Python 3 and works with Plone 5.2 and Plone 6.x.
-
-plone.restapi 8 does not officially support Python 3.6.
+plone.restapi 8 and 9 require Python 3 and work with Plone 5.2 and Plone 6.x.
```

Comment:
@mauritsvanrees @davisagli FYI: I updated the README to reflect our "more strict" policy regarding old Python versions.